Project21/Project21.sln

```diff
@@ -1,31 +1,31 @@
-﻿
-Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 17
-VisualStudioVersion = 17.10.35013.160
-MinimumVisualStudioVersion = 10.0.40219.1
-Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "Project21", "Project21\Project21.vcxproj", "{1D8458AB-8A7C-4448-9B0C-558F9AB54401}"
-EndProject
-Global
-	GlobalSection(SolutionConfigurationPlatforms) = preSolution
-		Debug|x64 = Debug|x64
-		Debug|x86 = Debug|x86
-		Release|x64 = Release|x64
-		Release|x86 = Release|x86
-	EndGlobalSection
-	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{1D8458AB-8A7C-4448-9B0C-558F9AB54401}.Debug|x64.ActiveCfg = Debug|x64
-		{1D8458AB-8A7C-4448-9B0C-558F9AB54401}.Debug|x64.Build.0 = Debug|x64
-		{1D8458AB-8A7C-4448-9B0C-558F9AB54401}.Debug|x86.ActiveCfg = Debug|Win32
-		{1D8458AB-8A7C-4448-9B0C-558F9AB54401}.Debug|x86.Build.0 = Debug|Win32
-		{1D8458AB-8A7C-4448-9B0C-558F9AB54401}.Release|x64.ActiveCfg = Release|x64
-		{1D8458AB-8A7C-4448-9B0C-558F9AB54401}.Release|x64.Build.0 = Release|x64
-		{1D8458AB-8A7C-4448-9B0C-558F9AB54401}.Release|x86.ActiveCfg = Release|Win32
-		{1D8458AB-8A7C-4448-9B0C-558F9AB54401}.Release|x86.Build.0 = Release|Win32
-	EndGlobalSection
-	GlobalSection(SolutionProperties) = preSolution
-		HideSolutionNode = FALSE
-	EndGlobalSection
-	GlobalSection(ExtensibilityGlobals) = postSolution
-		SolutionGuid = {670DCC48-7C38-4490-A9FC-CE8A5A72B3B9}
-	EndGlobalSection
-EndGlobal
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17

```

Comment:
Changes should not reflect in your solution file. Please use git di and git di main before you commit.